src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +74,21 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email", methods={"POST"})
+     * @Template()
+     */
+    public function resendRegistrationEmailAction(Request $request, UserManagerInterface $userManager, MailerInterface $mailer)
+    {
+        if($request->request->get("email") !== NULL) {
```

Comment:
There should be a space after `if`

Reply:
To check if the request contains a parameter, you can do `$request->request->has('email')`: it returns a boolean if the parameter exists or not.

Reply:
Please use simple quotes instead of double quotes